src/lua/api-gateway/tracking/validator/rewritingRulesValidator.lua

```diff
@@ -58,7 +58,7 @@ function _M:validate_rewrite_rules(config_obj)
         return nil
     end
 
-    ngx.var.rewritten_by = math.floor(tonumber(rewrite_rule.id)/100000)
+    ngx.var.rewritten_by = tonumber(rewrite_rule.id)
```

Comment:
I don't see what this has to be a number or why you need any modifications on this. I searched through out the code and I see no operations whatsoever. Are you sure you need it to be tonumber() isn't it showing in the logs either way?